source/databases-collections.txt

```diff
@@ -34,38 +34,35 @@ use the left navigation to:
 Databases and Collections
 -------------------------
 
-When you expand an active connection, |vsce| shows the databases in that
-deployment. Click a database to view the collections it contains.
+When you expand an active connection, |vsce| shows the databases in
```

Comment:
[suggestion] You could turn these two sentences into an unordered list:

- Click a database to view the collections it contains.
- Click a collection to view its documents, schema, and indexes.